fe/fe-core/src/main/java/com/starrocks/common/util/DynamicPartitionUtil.java

```diff
@@ -331,6 +344,22 @@ public static boolean isDynamicPartitionTable(Table table) {
                 tableProperty.getDynamicPartitionProperty().getEnable();
     }
 
+    public static boolean isTTLPartitionTable(Table table) {
+        if (!(table instanceof OlapTable) ||
+                !(((OlapTable) table).getPartitionInfo().getType().equals(PartitionType.RANGE))) {
+            return false;
+        }
+
+        TableProperty tableProperty = ((OlapTable) table).getTableProperty();
+        if (tableProperty == null || !tableProperty.getProperties().containsKey(PROPERTIES_PARTITION_TTL_NUMBER)) {
+            return false;
+        }
+
+        RangePartitionInfo rangePartitionInfo = (RangePartitionInfo) ((OlapTable) table).getPartitionInfo();
+        return rangePartitionInfo.getPartitionColumns().size() == 1;
```

Comment:
why the number of partition columns must be 1?

Reply:
Because we only support single-partition materialized views now, multi-partitions may be done later.